### PR TITLE
Fix FAQ's low-hanging fruit

### PR DESCRIPTION
**Pull request by jpmedley**
I know that we're trying to disperse this content and get rid of it. Since that is taking longer than expected, we need to fix its most egregious errors.

https://github.com/GoogleChromeLabs/Extension-Docs/issues/114

### CODE REVIEW

File: site/en/docs/extensions/mv3/faq/index.md [hunk 1]
```diff
@@ -109,53 +83,52 @@ requests.
...
+In the best case, these pages could not be updated as quickly as they are currently.
 In the worst case, it could mean that an update to one of these pages could cause an extension to
 break, causing key parts of the browser to stop working for users of that extension.
 
 The reason that [replacing the content][47] hosted at these URLs entirely is allowed is because it
 forces an extension developer to implement all of the functionality they want without depending on
 the browser's internal implementation to stay the same.
 
-### Can extensions open browser/page action popups without user interaction? {: #faq-open-popups }
+### Can extensions open browser/page action popups without a user interaction? {: #faq-open-popups }
```
Comment (AmySteam) [inline site/en/docs/extensions/mv3/faq/index.md:119]:
```suggestion
### Can extensions open action popups without a user interaction? {: #faq-open-popups }
```

File: site/en/docs/extensions/mv3/faq/index.md [hunk 2]
```diff
@@ -40,66 +40,40 @@ the browser.
...
 functions in your extensions without including JSON libraries in
 your code.
 
 ### Can extensions store data locally? {: #faq-dev-08 }
 
-Yes, extensions can use [localStorage][37] to store string data permanently. Using Chrome's built-in
-JSON functions, you can store complex data structures in localStorage. For extensions that need to
-execute SQL queries on their stored data, Chrome implements [client side SQL databases][38], which
-may be used as well.
+Yes. Extensions provides a [storage API][63] designed specifically for extenions. You can also use
```
Comment (AmySteam) [inline site/en/docs/extensions/mv3/faq/index.md:74]:
```suggestion
Yes. Extensions provide a [storage API][63] designed specifically for extensions. You can also use
```

**Comment by jpmedley** [comment]
PTAL

File: site/en/docs/extensions/mv3/faq/index.md [hunk 3]
```diff
@@ -40,66 +40,40 @@ the browser.
...
+the web platform's storage APIs: the [Web Storage API][37], and [IndexedDB][38]. If you use an
```
Comment (AmySteam) [inline site/en/docs/extensions/mv3/faq/index.md:75]: This may confuse developers in thinking they can use Window.localStorage in a service worker.

**Review by AmySteam — COMMENTED** [review]
Let a few comments

**Comment by 0569395312** [comment]
SA7180202461692222121016

احصل على Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by 569395312** [comment]
SA7180202461692222121016

**Comment by 0569395312** [comment]
SA7180202461692222121016

0569395312

في السبت، ٩ سبتمبر ٢٠٢٣, ١٢:١٧ ص 569395312 ***@***.***> كتب:

File: site/en/docs/extensions/mv3/faq/index.md [hunk 4]
```diff
@@ -40,66 +40,40 @@ the browser.
 
 ### Are extensions fetched from the web every time the browser is loaded? {: #faq-gen-03 }
 
-Extensions are downloaded by the Chrome browser upon install, and are subsequently run off of the
-local disk in order to speed up performance. However, if a new version of the extension is pushed
-online, it will be automatically downloaded in the background to any users who have the extension
-installed. Extensions may also make requests for remote content at any time, in order to interact
-with a web service or pull new content from the web.
+Extensions are downloaded by the Chrome browser upon install, and are subsequently run off of
+local memory to speed up performance. However, if a new version of the extension is uploaded to
```
Comment (jpmedley) [inline site/en/docs/extensions/mv3/faq/index.md:44]:
```suggestion
local disk to speed up performance. However, if a new version of the extension is uploaded to
```

File: site/en/docs/extensions/mv3/faq/index.md [hunk 5]
```diff
@@ -40,66 +40,40 @@ the browser.
...
 
 ### Can extensions store data locally? {: #faq-dev-08 }
 
-Yes, extensions can use [localStorage][37] to store string data permanently. Using Chrome's built-in
-JSON functions, you can store complex data structures in localStorage. For extensions that need to
-execute SQL queries on their stored data, Chrome implements [client side SQL databases][38], which
-may be used as well.
+Yes. Extensions provide a [storage API][63] designed specifically for extensions. You can also use
+the web platform's storage APIs: the [Web Storage API][37], and [IndexedDB][38]. If you use an
+[extension service worker][64], [CacheStorage][65] is also available.
```
Comment (jpmedley) [inline site/en/docs/extensions/mv3/faq/index.md:76]:
```suggestion
[extension service worker][64], you can use[CacheStorage][65] in addition to extensions APIs.
```

**Comment by jpmedley** [comment]
PTAL

**Review by AmySteam — APPROVED** [review]
LGTM 👍

**Pull request merged**